### PR TITLE
Yaz 23/terminate the entities services

### PR DESCRIPTION
**Pull request by juanDuqueS**
What: finished the rest of the services for each one of the entities

why: to have the connection with de database and the endpoints

Yaz 23/terminate the entities services

### CODE REVIEW

**Review by prometteo14 — APPROVED** [review]

File: back-end-yaz/src/server/services/UserService.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      }
+      return null;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async getUserById(id) {
+    try {
+      const theUser = await database.User.findOne({
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/services/UserService.js:39]: theUser isn't a good var name, please change it.

File: back-end-yaz/src/server/services/Payment_itemsService.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      }
+      return null;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async getPayment_itemsById(id) {
+    try {
+      const thePayment_items = await database.Payment_items.findOne({
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/services/Payment_itemsService.js:39]: thePayment_items isn't a good var name, please change it.

File: back-end-yaz/src/server/services/Payment_serviceService.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      }
+      return null;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async getPayment_serviceById(id) {
+    try {
+      const thePayment_service = await database.Payment_service.findOne({
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/services/Payment_serviceService.js:39]: thePayment_service isn't a good var name, please change it.

File: back-end-yaz/src/server/services/ServiceService.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      }
+      return null;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async getServiceById(id) {
+    try {
+      const theService = await database.Service.findOne({
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/services/ServiceService.js:39]: theService isn't a good var name, please change it.

Comment (juanDuqueS) [reply back-end-yaz/src/server/services/ServiceService.js]: Thanks, i already changed the variable name

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Pull request merged**